.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: c
 cache: ccache
+dist: trusty
```

Comment:
Can one choose xenial? Thing is that it comes with gcc-5 and clang-3.8, so that one doesn't waste time on installing them on per-test basis (which does take time)... This implies that one should omit even clang-3.6 from apt: section. One can argue that test with gcc-4 is meaningful. Yes, but maybe it's better to install gcc-4.8, conditionally would be best...